libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigation.java

```diff
@@ -387,6 +404,26 @@ public void startNavigation(@NonNull DirectionsRoute directionsRoute, @NonNull D
    */
   public void stopNavigation() {
     Timber.d("MapboxNavigation stopNavigation called");
+    isActiveGuidanceOnGoing.set(false);
+    if (isFreeDriveEnabled.get()) {
+      enableFreeDrive();
+    }
+    stopNavigationService();
+  }
+
+  private void killNavigation() {
+    Timber.d("MapboxNavigation killNavigation called");
```

Comment:
Leftover

Reply:
I added this for consistency with `stopNavigation` 👀 https://github.com/mapbox/mapbox-navigation-android/blob/6a934f45a81d44f6d7697c9750c0f3a4027ca4f9/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigation.java#L406 I can remove, though, what do you think?

Reply:
I added this for consistency with `stopNavigation` 👀 https://github.com/mapbox/mapbox-navigation-android/blob/6a934f45a81d44f6d7697c9750c0f3a4027ca4f9/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigation.java#L406 I can remove, though, what do you think?

Reply:
Let's move `Timber.d("MapboxNavigation stopNavigation called"); ` to `stopNavigationService()` method then. I don't think `killNavigation` provides any additional information.